doc/source/operations/rocky-linux-9.rst

```diff
@@ -0,0 +1,517 @@
+==========================
+Migrating to Rocky Linux 9
+==========================
+
+Overview
+========
+
+This document describes how to migrate systems from CentOS Stream 8 to Rocky Linux 9.
+This procedure must be performed on CentOS Stream 8 OpenStack Yoga systems before it is possible to upgrade to OpenStack Zed.
+It is possible to perform a rolling migration to ensure service is not disrupted. This section covers the steps required to perform such a migration.
+
+For hosts running CentOS 8 Stream, the migration process has a simple structure:
+
+- Remove a CentOS Stream 8 host from service
+- Reprovision the host with a Rocky Linux 9 image
+- Configure and deploy the host with Rocky Linux 9 containers
+
+While it is technically possible to migrate hosts in any order, it is strongly recommended that migrations for one type of node are completed before moving on to the next i.e. all compute node migrations are performed before all storage node migrations.
+
+The order of node groups is less important, however it is arguably safest to perform controller node migrations first, given that they are the most complex and it is easiest to revert their state in the event of a failure.
+This guide covers the following types of hosts:
+
+- Controllers
+- Compute hosts
+- Storage hosts
+- Seed
+- Seed hypervisor
```

Comment:
I think it would be best to separate out the TODO host types up here since this implies the guide is finalised.